### PR TITLE
Fix install and uninstall from .terraform-version in path with spaces

### PR DESCRIPTION
**Pull request by iamhsa**
Correction proposal for #107

### CODE REVIEW

File: libexec/tfenv-uninstall [hunk 1]
```diff
@@ -7,21 +7,14 @@ source ${TFENV_ROOT}/libexec/helpers
...
 
-if [ -z "${1}" ]; then
-  version_file="$(tfenv-version-file)"
-  if [ "${version_file}" != "${TFENV_ROOT}/version" ];then
-    version_requested="$(cat ${version_file} || true)"
-  fi
-else
-  version_requested="${1}"
-fi
+get_version_requested "${1}"
```
Comment (Zordrak) [inline libexec/tfenv-uninstall]: I'm not sure it's a good idea to invoke a common function that uses tfenv-min-required in `uninstall`

File: libexec/tfenv-uninstall [hunk 2]
```diff
@@ -7,21 +7,14 @@ source ${TFENV_ROOT}/libexec/helpers
...
 
 if [[ "${version_requested}" =~ ^latest\:.*$ ]]; then
   version="${version_requested%%\:*}"
   regex="${version_requested##*\:}"
 elif [[ "${version_requested}" =~ ^latest$ ]]; then
   version="${version_requested}"
-  regex=""
+  regex="^[0-9]\+\.[0-9]\+\.[0-9]\+$"
```
Comment (Zordrak) [inline libexec/tfenv-uninstall]: Not sure the value of adding this here other than to restrict possibilities. Regex only exists for the purpose of a user specifying one.

File: libexec/helpers [hunk 3]
```diff
@@ -24,3 +24,27 @@ function curlw () {
 
   curl ${TLS_OPT} "$@"
 }
+
+function get_version_requested () {
+  if [ -z "${1}" ]; then
+    version_file="$(tfenv-version-file)"
+    if [ "${version_file}" != "${TFENV_ROOT}/version" ];then
+      version_requested="$(cat "${version_file}" || true)"
```
Comment (Zordrak) [inline libexec/helpers]: Is it fair to say that the only change that specifically intends to fix the spaces issue is these quotes around the version file? This seems perfectly appropriate.

**Comment by Zordrak** [comment]
Proposing to close this since #111 should provide the resolution

File: libexec/helpers (hunk 3 above)
Comment (iamhsa) [reply libexec/helpers]: You're right, quotation marks are the only important thing. 
I simply wanted to share the code between install and uninstall.

File: libexec/tfenv-uninstall (hunk 1 above)
Comment (iamhsa) [reply libexec/tfenv-uninstall]: > Proposing to close this since #111 should provide the resolution
It seems to be a good idea.

**Comment by Zordrak** [comment]
Fixed by #111

**Pull request closed without merging**